src/sandstorm/web-session.capnp

```diff
@@ -140,6 +145,9 @@ interface WebSession @0xa50711a14d35a8ce extends(Grain.UiSession) {
     struct Header {
       name @0 :Text;  # lower-cased name
       value @1 :Text;
+      prefixed @2 :Bool = false;
```

Comment:
In fact, as-is, this is exploitable: Existing apps compiled before this change will not check the `prefix` field, so after this change anyone could send `X-Sandstorm-App-X-Sandstorm-User-Id` to old apps to falsify authentication info.